pyspark-notebook/Dockerfile

```diff
@@ -64,9 +64,10 @@ RUN fix-permissions "/etc/ipython/"
 
 USER ${NB_UID}
 
-# Install pyarrow
+# Install pyarrow and pin pandas to version 1.5.3 see #1924
 RUN mamba install --yes \
-    'pyarrow' && \
+    'pyarrow' \
+    'pandas=1.5.3' && \
```

Comment:
I think a better way would be to install >=1.5.3 and <2.0.0, and then actually pin it (so it won't be updated in the `all-spark-notebook`).